proj/src/main/webapp/index.html

```diff
@@ -5,42 +5,17 @@
         <meta charset="UTF-8">
         <title>My Portfolio</title>
         <link rel="stylesheet" href="style.css">
-        
-        <!-- The core Firebase JS SDK is always required and must be listed first -->
-        <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-app.js"></script>
-
-        <!-- TODO: Add SDKs for Firebase products that you want to use
-            https://firebase.google.com/docs/web/setup#available-libraries -->
-        <script src="https://www.gstatic.com/firebasejs/7.15.4/firebase-analytics.js"></script>
-
-        <script>
-            var firebaseConfig = {
-            apiKey: "AIzaSyAcpxnpwrTCO4XTymTcneRscMBzJBne2Qg",
-            authDomain: "arringtonh-step-2020-d.firebaseapp.com",
-            databaseURL: "https://arringtonh-step-2020-d.firebaseio.com",
-            projectId: "arringtonh-step-2020-d",
-            storageBucket: "arringtonh-step-2020-d.appspot.com",
-            messagingSenderId: "336825043126",
-            appId: "1:336825043126:web:1256d7b08f1c8daa93be17",
-            measurementId: "G-VJQS6TEYGV"};
-        </script>
-
```

Comment:
Are you intentionally removing all of these?

Reply:
yeah i moved them to another file to clutter things up less, and so people could use their own configs when pulling

Reply:
i just realized i didn't mean to delete all of it so i had to add some back 🙃